tests/foreman/ui/test_location.py

```diff
@@ -524,10 +528,18 @@ def test_positive_remove_environment(self):
             for env_name in valid_env_names():
                 with self.subTest(env_name):
                     loc_name = gen_string('alpha')
-                    env = entities.Environment(name=env_name).create()
+                    env = entities.Environment(
+                        name=env_name,
+                        organization=[self.org_],
+                    ).create()
                     self.assertEqual(env.name, env_name)
-                    set_context(session, org=ANY_CONTEXT['org'])
-                    make_loc(session, name=loc_name, envs=[env_name])
+                    set_context(session, org=self.org_.name)
```

Comment:
we have 8 calls to set_context so we are going to save only 1 minute in this module..